### ISSUE TITLE
Dummy network

### ISSUE THREAD

**Issue by cswinter** [entity-neural-network/incubator#3]
Implement most simple possible PyTorch neural network that is initialized with an entity-gym state space and action space.
Forward pass is a single linear transformation for each entity to project them into the same embedding space, and then indexes with actions masks to select the right entities and apply action heads.
